### PR TITLE
tests

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

File: src/Eye4web/ZfcUser/Settings/Module.php [hunk 1]
```diff
@@ -16,9 +16,6 @@ class Module
 {
     public function onBootstrap(MvcEvent $e)
     {
-        $eventManager        = $e->getApplication()->getEventManager();
-        $moduleRouteListener = new ModuleRouteListener();
-        $moduleRouteListener->attach($eventManager);
```
Comment (samsonasik) [inline src/Eye4web/ZfcUser/Settings/Module.php:21]: removed as ModuleRouteListener can only be called in Application module.

File: src/Eye4web/ZfcUser/Settings/Factory/Mapper/DoctrineORM/UserSettingMapperFactory.php [hunk 2]
```diff
@@ -32,7 +32,7 @@ class UserSettingMapperFactory implements FactoryInterface
      */
     public function createService(ServiceLocatorInterface $serviceLocator)
     {
-        $entityManager = $serviceLocator->get('zfcuser_doctrine_em');
+        $entityManager = $serviceLocator->get('Doctrine\ORM\EntityManager');
```
Comment (samsonasik) [inline src/Eye4web/ZfcUser/Settings/Factory/Mapper/DoctrineORM/UserSettingMapperFactory.php]: Changed passed Service Name to <code>Doctrine\ORM\EntityManager</code> as there is no service named <code>zfcuser_doctrine_em</code>

Comment (samsonasik) [inline src/Eye4web/ZfcUser/Settings/Factory/Mapper/DoctrineORM/UserSettingMapperFactory.php]: oh, I know now, there is <code>zfcuser_doctrine_em</code> in <code>ZfcUserDoctrineORM</code>, I will add it at composer.json at require-dev to support test.

Comment (samsonasik) [inline src/Eye4web/ZfcUser/Settings/Factory/Mapper/DoctrineORM/UserSettingMapperFactory.php]: oh, the "zf-commons/zfc-user-doctrine-orm" addition should be in "require" in composer.json

File: composer.json [hunk 3]
```diff
@@ -23,7 +23,8 @@
     },
     "require": {
         "php": "~5.4",
-        "zf-commons/zfc-user": "~1.2"
+        "zf-commons/zfc-user": "~1.2",
+        "zf-commons/zfc-user-doctrine-orm": "1.*"
```
Comment (samsonasik) [inline composer.json:27]: added to support "zfcuser_doctrine_em" service call and tests.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+before_script:
+  - mkdir -p build/logs
+  - composer self-update
+  - composer install --prefer-source
+
+script:
+  - ./vendor/bin/phpunit --coverage-clover ../build/logs/clover.xml
+  - ./vendor/bin/phpcs --standard=PSR2 ./src/ ./test/ZfcUserSettingsTest/ -n
+  - ./vendor/bin/test-reporter --stdout > codeclimate.json
+  - "curl -X POST -d @codeclimate.json -H 'Content-Type: application/json' -H 'User-Agent: Code Climate (PHP Test Reporter v1.0.1-dev)' https://codeclimate.com/test_reports"
```
Comment (samsonasik) [inline .travis.yml:15]: added <code>.travis.yml</code> config, need repo token in travis if using test-reporter.

File: src/Eye4web/ZfcUser/Settings/View/Helper/UserSettingHelper.php [hunk 5]
```diff
@@ -30,11 +31,17 @@ class UserSettingHelper extends AbstractHelper
...
+    private $zfcUserIdentity;
+
+    /**
      * @param UserSettingsServiceInterface $userSettingsService
      */
-    public function __construct(UserSettingsServiceInterface $userSettingsService)
+    public function __construct(UserSettingsServiceInterface $userSettingsService, ZfcUserIdentity $zfcUserIdentity)
     {
         $this->userSettingsService = $userSettingsService;
+        $this->zfcUserIdentity     = $zfcUserIdentity;
```
Comment (samsonasik) [inline src/Eye4web/ZfcUser/Settings/View/Helper/UserSettingHelper.php:44]: refactored to make easier to call __invoke in test.

**Comment by samsonasik** [comment]
Update coverage : 
coverage-zfcusersettings
I sure the <code>View\Helper\UserSettingHelper::getSetting($setting, null)</code> should called with dataProvider, but don't know why calling <code>__invoke() </code> in <code>zfcUserIdentity</code> view helper doesn't get the coverage executed.

**Comment by samsonasik** [comment]
Finally, code coverage 100% \m/ , I've updated <code>View\Helper\UserSettingHelper</code> test.
100_percent_coverage

**Comment by Danielss89** [comment]
perfect!

**Pull request merged**